sally.yaml

```diff
@@ -5,6 +5,8 @@ url: go.uber.org
 packages:
     atomic:
         repo: github.com/uber-go/atomic
+    cadence:
```

Comment:
Nope, no preference just wanted to make sure you were making this decision consciously. Your reasoning seems to make sense to me, since most people will just want to use the client library